front/app/components/admin/PostManager/components/PostPreview/Idea/Components/ListItemWithBullet.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+import { List } from 'semantic-ui-react';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+type Props = {
+  message: { id: string; defaultMessage: string };
+};
+const ListItemWithBullet = ({ message }: Props) => {
+  return (
+    <List.Item>
```

Comment:
Let's get rid of this and just replace it by a `li` element? Would be great to not add any more dependencies on `semantic-ui`